packages/vue/src/VueTolgee.ts

```diff
@@ -1,17 +1,21 @@
 import type { App } from 'vue';
-import { ref } from 'vue';
+import { ref, watch } from 'vue';
 import {
   getTranslateProps,
   TolgeeInstance,
   TFnType,
   DefaultParamType,
   TranslationKey,
 } from '@tolgee/web';
+import { TolgeeVueContext } from './types';
 
 type Options = {
   tolgee?: TolgeeInstance;
+  isSSR?: boolean;
```

Comment:
This name might be a bit confusing, since `isSSR` is often used when distinguising if we are on the server or on the client. I'd maybe name it e.g. `enableSSR` or something

Reply:
yes, totally agreed